components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/listeners/GatewayStartupListener.java

```diff
@@ -84,7 +93,14 @@ public void completedServerStartup() {
                 new APIMgtGatewayCacheMessageListener());
         jmsTransportHandlerForEventHub
                 .subscribeForJmsEvents(APIConstants.TopicNames.TOPIC_NOTIFICATION, new GatewayJMSMessageListener());
+    }
 
```

Comment:
As discussed offline let's limit the retries and proceed with startup in case of a failure.

Reply:
https://github.com/wso2/carbon-apimgt/pull/8945/commits/ae40fd8c53d3812a64ea9c97d69bb8600ff81a52